src/themes/shared/recipes/spinner.ts

```diff
@@ -0,0 +1,29 @@
+import { defineRecipe } from '@chakra-ui/react';
+
+export const spinnerRecipe = defineRecipe({
+  className: 'chakra-spinner',
+  base: {
+    display: 'inline-block',
+    borderColor: 'gray.900',
+    border: '2px',
+    borderRadius: 'full',
+    width: 'var(--spinner-size)',
+    height: 'var(--spinner-size)',
+    animation: 'spin',
+    '--spinner-track-color': 'transparent',
+    borderBottomColor: 'var(--spinner-track-color)',
+    borderInlineStartColor: 'var(--spinner-track-color)',
+  },
+  variants: {
+    size: {
+      inherit: { '--spinner-size': 'sizes.sm' },
```

Comment:
This looks off. Shouldn't `inherit` be the current font size of something along those lines?